Form/ChangePassword.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Form;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class ChangePassword
+{
+    /**
+     * @Assert\Length(
+     *     min = 6,
+     *     minMessage = "sumocoders.multiuserbundle.form.length"
+     * )
+     */
+    protected $newPassword;
+
+    public function getNewPassword()
+    {
+        return $this->newPassword;
+    }
+
+    public function setNewPassword($password)
+    {
+        $this->newPassword = $password;
+    }
```

Comment:
als dit puur een data transfer object is voor je form kan je evengoed met een public property werken en de getters en setters weglaten.